### ISSUE TITLE
Black theme: Transparent context menu.

### ISSUE THREAD

**Issue by kr428** [andstatus/andstatus#565]
In some(?) situations, it seems using the app with a black theme causes context menu to display with a transparent background, barely readable. It reliably happens here but I didn't yet completely figure out why... 😬

![Screenshot_20221223-155719_AndStatus](https://user-images.githubusercontent.com/685879/209355989-8bdcc9ac-8268-4de8-b645-88947972b8d2.jpg)

**Comment by yvolk** [comment]
Yes, I also notice this from time to time. Didn't figure out why yet...
